app/controllers/api/v1/topics_controller.rb

```diff
@@ -0,0 +1,14 @@
+module Api
+  module V1
+    class TopicsController < ApplicationController
+      include Concerns::ActAsApiRequest
+      include DeviseTokenAuth::Concerns::SetUserByToken
+
+      before_action :authenticate_user!
```

Comment:
Along with @mrodriguez90 comment, this should also go into `api_controller.rb`, so it requires authentication for all endpoints by default.